scripts/analytics/institution_summary.py

```diff
@@ -53,7 +50,7 @@ def get_events(self, date):
                     'name': institution.name,
                 },
                 'users': {
-                    'total': OSFUser.find(user_query).count(),
+                    'total': OSFUser.objects.filter(user_query).count(),
                 },
                 'nodes': {
                     'total': AbstractNode.find_by_institutions(institution, node_query).count(),
```

Comment:
Could all of these use the reverse relation? i.e. `institution.nodes.filter(node_query).count()`?